packages/mui-system/src/Grid/gridClasses.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  unstable_generateUtilityClass as generateUtilityClass,
+  unstable_generateUtilityClasses as generateUtilityClasses,
+} from '@mui/utils';
+
+export interface GridClasses {
+  /** Styles applied to the root element. */
+  root: string;
+  /** Styles applied to the root element if `container={true}`. */
+  container: string;
+  /** Styles applied to the root element if `direction="column"`. */
+  'direction-xs-column': string;
+  /** Styles applied to the root element if `direction="column-reverse"`. */
+  'direction-xs-column-reverse': string;
+  /** Styles applied to the root element if `direction="row-reverse"`. */
+  'direction-xs-row-reverse': string;
+  /** Styles applied to the root element if `wrap="nowrap"`. */
+  'wrap-xs-nowrap': string;
+  /** Styles applied to the root element if `wrap="reverse"`. */
+  'wrap-xs-wrap-reverse': string;
```

Comment:
```suggestion
  /** Class name applied to the root element. */
  root: string;
  /** Class name applied to the root element if `container={true}`. */
  container: string;
  /** Class name applied to the root element if `direction="column"`. */
  'direction-xs-column': string;
  /** Styles applied to the root element if `direction="column-reverse"`. */
  'direction-xs-column-reverse': string;
  /** Class name applied to the root element if `direction="row-reverse"`. */
  'direction-xs-row-reverse': string;
  /** Class name applied to the root element if `wrap="nowrap"`. */
  'wrap-xs-nowrap': string;
  /** Class name applied to the root element if `wrap="reverse"`. */
  'wrap-xs-wrap-reverse': string;
```

Reply:
🤔 Should we keep it like this for consistency? Joy is using `Styles` as well.